src/ir_Corona.h

```diff
@@ -0,0 +1,148 @@
+// Corona A/C
+//
+// Copyright 2020 Christian Nilsson
+
+// Supports:
+//   Brand: Corona,  Model: CSH-N2211 A/C
+//   Brand: Corona,  Model: CSH-N2511 A/C
+//   Brand: Corona,  Model: CSH-N2811 A/C
+//   Brand: Corona,  Model: CSH-N4011 A/C
+//   Brand: Corona,  Model: AR-01 remote
+//
+// Ref: https://docs.google.com/spreadsheets/d/1zzDEUQ52y7MZ7_xCU3pdjdqbRXOwZLsbTGvKWcicqCI/
+// Ref: https://www.corona.co.jp/box/download.php?id=145060636229
+
+#ifndef IR_CORONA_H_
+#define IR_CORONA_H_
+
+#define __STDC_LIMIT_MACROS
+#include <stdint.h>
+#ifndef UNIT_TEST
+#include <Arduino.h>
+#endif
+#include "IRremoteESP8266.h"
+#include "IRsend.h"
+#ifdef UNIT_TEST
+#include "IRsend_test.h"
+#endif
+
+// Constants
+
+// CORONA_AC
+const uint8_t kCoronaAcSectionBytes = 7;  // kCoronaAcStateLengthShort
+const uint8_t kCoronaAcSections = 3;
+const uint8_t kCoronaAcSectionHeader0Pos = 0;
+const uint8_t kCoronaAcSectionHeader0 = 0x28;
+const uint8_t kCoronaAcSectionHeader1Pos = 1;
+const uint8_t kCoronaAcSectionHeader1 = 0x61;
+const uint8_t kCoronaAcSectionLabelPos = 2;
+const uint8_t kCoronaAcSectionLabelBase = 0x0D;  // 0b1101
+const uint8_t kCoronaAcSectionData0Pos = 3;
+const uint8_t kCoronaAcSectionData0InvPos = 4;
+const uint8_t kCoronaAcSectionData1Pos = 5;
+const uint8_t kCoronaAcSectionData1InvPos = 6;
+const uint8_t kCoronaAcSectionData0Base = 0x10;  // D0 Pos 4 always on
+
+const uint8_t kCoronaAcSettingsSection = 0;
+// D0
+const uint8_t kCoronaAcFanOffset = 0;        // D0 LSB Pos 0-1
+const uint8_t kCoronaAcFanSize = 2;
+const uint8_t kCoronaAcFanAuto =    0b00;  // 0
+const uint8_t kCoronaAcFanLow =     0b01;  // 1
+const uint8_t kCoronaAcFanMedium =  0b10;  // 2
+const uint8_t kCoronaAcFanHigh =    0b11;  // 3
+
+// One bit unknown                           // D0 Pos 2
+const uint8_t kCoronaAcPowerSaveOffset = 3;  // D0 Pos 3
+// One bit unknown always on                 // D0 Pos 4
+// One bit unknown                           // D0 Pos 5
+const uint8_t kCoronaAcSwingVToogleOffset = 6;  // D0 Pos 6
```

Comment:
Still got the typo `Toogle` -> `Toggle`

Reply:
a, sorry missunderstood, and didn't read, thank you